homeassistant/components/todo/strings.json

```diff
@@ -6,6 +6,16 @@
     }
   },
   "services": {
+    "list_items": {
+      "name": "List to-do list items",
+      "description": "List items on a to-do list.",
+      "fields": {
+        "status": {
+          "name": "Status",
+          "description": "Only return to-do items with the specified status."
+        }
```

Comment:
Should this specify that the default, if not specified, is to only include items with a `needs_action` status?

Reply:
Yes, agreed, done.